editorSupport/sublime-reason/Reason.py

```diff
@@ -28,7 +28,6 @@ def reason_command_line(file_name):
 
     return [
         refmt,
-        '--use-stdin', 'true',
         '--parse', 're',
         '--interface', is_interface(file_name),
```

Comment:
Interesting. I think you don't need to fix it for emacs since it always copies the buffer into a temporary file before formatting.